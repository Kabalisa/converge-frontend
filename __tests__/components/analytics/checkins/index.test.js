import React from 'react';
import { shallow } from 'enzyme';
import Checkins from '../../../../src/components/analytics/checkins';

describe('Checkins component', () => {
  const wrapper = shallow(<Checkins />);

  it('should render properly', () => {
    expect(wrapper).toMatchSnapshot();
  });
});

