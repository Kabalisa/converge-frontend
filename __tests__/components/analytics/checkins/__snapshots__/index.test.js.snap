// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Checkins component should render properly 1`] = `
<div
  className="checkins"
>
  <DonutChart
    chartSvg="test-file-stub"
    chartTitle="% of Checkins"
    entries={15}
    hasInfo={true}
    hintText="% of Checkins pie chart"
    percentage={75}
    total={20}
  />
  <DonutChart
    chartSvg="test-file-stub"
    chartTitle="% of App Bookings"
    entries={4}
    hasInfo={true}
    hintText="% of App Bookings pie chart"
    percentage={20}
    total={20}
  />
  <DonutChart
    chartSvg="test-file-stub"
    chartTitle="Average Room Utilisation"
    entries={16}
    hasInfo={false}
    hintText="Average Room Utilisation pie chart"
    percentage={80}
    total={20}
  />
</div>
`;
